app/controllers/users_controller.rb

```diff
@@ -126,6 +126,7 @@ def finish_successful_login(profile)
     # Stores additional information in the session pertaining to the user
     store_profile(profile)
     Rails.logger.debug "Successful URS Login by user #{authenticated_urs_uid}" if session[:login_method] == 'urs'
```

Comment:
is this not true for proposal mode?
If the new line is necessary, then I would combine it to the end of this line: `|| Rails.configuration.proposal_mode`

Reply:
Different logging levels. I assumed we don't have debug on in SIT+

Reply:
ok